nautobot/extras/models/customfields.py

```diff
@@ -255,6 +255,10 @@ class Meta:
     def __str__(self):
         return self.label or self.name.replace("_", " ").capitalize()
 
+    @property
+    def slug(self):
```

Comment:
Possibly add a docstring here explaining that name will eventually be replaced with slug?